src/types/models/User.ts

```diff
@@ -26,8 +26,6 @@ export const User = objectType({
     t.model.updatedAt();
     t.model.deletedAt();
     t.model.posts({ pagination: false });
-    t.model.profile({
-      type: 'Profile',
-    });
+    t.model.profile();
```

Comment:
@smallbee3 Looks like some changes been made internally. This won't let us in current project either after upgrading packages.